src/main/webapp/app/exercises/shared/result/result-detail.component.ts

```diff
@@ -126,16 +123,16 @@ export class ResultDetailComponent implements OnInit {
 
     numberOfAggregatedTestCases = 0;
 
-    // Icons
-    faCircleNotch = faCircleNotch;
-    faExclamationTriangle = faExclamationTriangle;
+    feedbackService: FeedbackService;
 
     constructor(
         public activeModal: NgbActiveModal,
         private resultService: ResultService,
         private buildLogService: BuildLogService,
         private translateService: TranslateService,
         private profileService: ProfileService,
+        private feedbackServiceImpl: FeedbackServiceImpl,
```

Comment:
`feedbackServiceImpl` contains the generic functions for all feedback types. Sadly we cant just have a single interface `FeedbackService`, which is set and contains all relevant methods as angular dependency injection does not play nice with inheritance